examples/peer/contracts/PeerFactory.sol

```diff
@@ -0,0 +1,46 @@
+/*
+  Copyright 2019 Swap Holdings Ltd.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+pragma solidity 0.5.10;
+pragma experimental ABIEncoderV2;
+
+import "@airswap/peer/contracts/Peer.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract PeerFactory {
+
+  event NewTrustedPeer(address peer, address swap, address owner);
+
+  mapping(address => bool) public trustedPeers;
```

Comment:
why do we need to keep track of `trustedPeers`?

Reply:
The purpose of this factory is so that we can allow people to deploy peers that we know are trustworthy. But by that logic we should therefore have a way of quickly checking if the peers came out of the factory or not.

Reply:
factoryCreatedPeers instead

Reply:
also why not use a mapping of Peers -> bool

Reply:
Per meeting today the mapping will be more generally named after a whitelist

Reply:
Will make mapping `IPeer` to `Bool`

Reply:
Looking at this again, I think that it may make it simpler to leave the mapping as address => bool, so that anyone can easily look up an address in the mapping without having to cast the address to a Peer. Thoughts?

Reply:
i think that would work, given we're changing the purpose in that this function will be called not just for Peers but for external addresses as well.